tests/test_fixup_scripts.py

```diff
@@ -187,3 +190,28 @@ def test_fixup_script_different_version(self):
         assert clone_path in data
 
         assert clone_path + '/bin/python2' in data
+
+    def test_fixup_pth_file(self):
+        '''Prior to version 0.2.7, the method fixup_pth_file() wrongly wrote unicode files'''
+
+        content = os.linesep.join(('/usr/local/bin/someFile', '/tmp/xyzzy', '/usr/local/someOtherFile', ''))
+        pth = '/tmp/test_fixup_pth_file.pth'
+
+        with open(pth, 'w') as fd:
+            fd.write(content)
+
+        # Find size of file - should be plain ASCII/UTF-8...
+
+        orgStat = os.stat(pth)
```

Comment:
Can you please change `orgStat` to `stat_result`?

Reply:
I will change from camelCase to snake_case...:-)